layouts/partials/new-google-analytics.html

```diff
@@ -0,0 +1,41 @@
+{{- $pc := .Site.Config.Privacy.GoogleAnalytics -}}
+{{- if not $pc.Disable -}}
+{{ with .Site.GoogleAnalytics }}
+<script type="application/javascript">
+{{ template "__ga_js_set_doNotTrack" $ }}
+if (!doNotTrack) {
+	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+	{{- if $pc.UseSessionStorage }}
+	if (window.sessionStorage) {
+		var GA_SESSION_STORAGE_KEY = 'ga:clientId';
+		ga('create', '{{ . }}', {
```

Comment:
Do we need to add the other properties here, too?

Reply:
They're present in line 25, per the Google Analytics help article.

Reply:
It looks like there's some templating logic in here, though. One case is if local storage is enabled, the other if it is not (assuming I'm reading this right). I imagine we want to track both cases?

Reply:
That I don't have an answer to. I just went to how Hugo implements their GA code, copied that to a new file that I could actually see/modify as opposed to being buried in a black box deep in some guts somewhere, and added the other properties. If we want to fiddle with the logic, that is out of scope for me :)